tests/logs/publish/test_publish/test_repo_publish_cache_cleanup.txt

```diff
@@ -0,0 +1,13 @@
+[    INFO] run publish
+[    INFO] Get repos: started
+[    INFO] Get repos: finished
+[    INFO] Publish: started
+[    INFO] Publishing repo1
+[    INFO] Publishing repo1 successful
```

Comment:
This log shows up the complaint I raised earlier, about having identical log messages in sequence for a repo.  This doesn't make any sense for someone reading the log.